packages/components/date-picker/src/composables/use-basic-date-table.ts

```diff
@@ -133,9 +133,7 @@ export const useBasicDateTable = (
     const shouldIncrement = setDateText(cell, { count, rowIndex, columnIndex })
 
     const cellDate = cell.dayjs!.toDate()
-    cell.selected = _selectedDate.find(
-      (d) => d.valueOf() === cell.dayjs!.valueOf()
-    )
+    cell.selected = _selectedDate.find((d) => d.isSame(cell.dayjs, 'dates'))
```

Comment:
```suggestion
    cell.selected = _selectedDate.find((d) => d.isSame(cell.dayjs, 'day'))
```

Reply:
Wouldn't it be better to use `date`?

Reply:
I asked the author of dayjs, `date`, `dates`, `day`, `days` are the same. In order to be compatible with momentjs. So, choose a semantically explicit value is better.